tests/sftp/test_sftp_server.py

```diff
@@ -93,10 +93,6 @@ def __run_server(self):
 
                 while transport.is_active():
                     time.sleep(1)
-            except ConnectionAbortedError:
-                pass
-            except Exception as e:
-                logger.critical(e, exc_info=True)
```

Comment:
This change is because I discovered that the `except Exception as e:` block was being executed occasionally. This caused the project coverage to fluctuate, which in turn led to failures in our CI coverage checks.

Considering this is only test code and does not affect users, we shouldn't blindly catch such a broad exception. By not catching this exception, future CI failures will reveal the actual traceback, allowing us to understand why this line of code is being executed intermittently.